modules/doc/content/getting_started/installation/conda.md

```diff
@@ -5,7 +5,7 @@ application development is via Conda's myriad array of libraries. Follow these
 instructions to create an environment on your machine using Conda. At this time,
 an option to install MOOSE directly on a Windows system is not yet supported.
 On-going efforts are being made to add a conda installation option for Windows,
-and an experimental [WSL](installation/windows10.md) option is available.
+and an experimental [WSL](installation/windows.md) option is available.
```

Comment:
Unless you want to fix other application's documentation, I suggest leaving it windows10...

Reply:
Since windows 11 is out, I would rather just make the name generic while we have the moment to change things around.